ui/src/main/js/common/component/Logo.js

```diff
@@ -4,7 +4,7 @@ import '../../../css/logos.scss';
 
 const Logo = () => (
     <div className="productLogo">
-        <span className="synopsysLogoSpan">
+        <span className="blackduckLogoSpan">
             <img
                 className="synopsysHeaderLogo"
                 src="https://www.synopsys.com/content/dam/synopsys/company/about/legal/synopsys-logos/whitelogo/synopsys_wht.png"
```

Comment:
Doesn't this need to get removed?  Maybe out of scope for this PR.

Reply:
Yeah out of scope for this PR. I was planning to tackle this on the spike ticket, IALERT-3704.